test/e2e/apps/statefulset.go

```diff
@@ -1515,7 +1516,8 @@ var _ = SIGDescribe("StatefulSet", func() {
 
```

Comment:
this test doesn't even `waitForStatus`, even though it probably never flakes since the number of replicas changes. It might be worth to use the same approach here as well.

Reply:
ack